server/scratchy_server/resources/roomres.py

```diff
@@ -31,7 +32,12 @@ def delete(self, roomId):
 class AllRoomRes(MethodResource):
 
     @marshal_with(AllRoomSchema, code=200)
-    def get(self):
+    @use_kwargs({"userId": fields.String()}, location="query")
+    def get(self, userId=None):
+        if userId != None: # Search room by containing a specific userid
```

Comment:
userId is not None
see on http://jaredgrubb.blogspot.com/2009/04/python-is-none-vs-none.html

Reply:
Corrected 526db31